eg3.py

```diff
@@ -0,0 +1,31 @@
+"""You are given a list of sentences. Create a word tree from each sentence and find how many times the word is appeared in other word trees."""
+
+#list created with sentences
+sentences = ["My name is Ram", "He is a good person", "You should be careful while coding",
+             "He can do better", "The person is mysterious", "Jay Shree Ram", "It is my pen"]
+
+
+split_sentences = []
+for sentence in sentences:
+    split_sentences.append(sentence.split())  #spliting each sentence of list
```

Comment:
You can use .extend to avoid for loop below. Please check that.

Reply:
yes, i used .extend method insted of below for loop(see line no.14)